src/pp-annotator/viewer/PrzypisViewer.tsx

```diff
@@ -209,21 +232,23 @@ export default class PrzypisViewer extends Widget {
     //            opposed to merely mousing off the current one). Default: false
     //
     // Returns a Promise.
-    _startHideTimer = (activity) => {
+    _startHideTimer = (activity: boolean) => {
 
         /*todo KG
         This part is copied straight from annotator.Viewer and might not be very consistent with other code;
         We should consider refactoring it and making it more explicit if we need to modify it
         */
 
-        if (typeof activity === 'undefined' || activity === null) {
+        if (typeof activity === 'undefined' || activity === null || activity === false) {
```

Comment:
`!activity` ?

Reply:
chociaż nie wiem po co do falsy przypisywać `false`, może dałoby się logikę niżej oprzeć na `falsy` ?

Reply:
jako że to skopiowane z annotatora i dosyć skomplikowane to tego nie refaktorowałem bez testów :D

Reply:
tak, ten fragment może wylecieć całkiem bo już wymuszone żeby to był `boolean` anyway